tests/unit/src/main/scala/scala/meta/internal/metals/debug/TestDebugger.scala

```diff
@@ -89,7 +89,7 @@ final class TestDebugger(
   }
 
   def disconnect: Future[Unit] = {
-    ifNotFailed(debugger.disconnect)
+    ifNotFailed(debugger.disconnect).map(_ => terminated.trySuccess(()))
```

Comment:
It seems that sometimes we might not get the terminated message, so this is to work around https://github.com/scalacenter/scala-debug-adapter/issues/66